sql/executor.go

```diff
@@ -99,6 +99,11 @@ func (e *Executor) Execute(args driver.Request) (driver.Response, int, error) {
 		}
 		planMaker.setTxn(txn, planMaker.session.Txn.Timestamp.GoTime())
 	}
+	var err error
+	planMaker.evalCtx.Location, err = getTimeZone(planMaker.session)
```

Comment:
I think now that EvalContext has a Location instead of the two separate fields that it's better to just eagerly load the time zone instead of a lazy accessor. (We can revisit this if it turns out to be slow, but I would expect `time.LoadLocation()` to be cached and if it's not we can add a cache of our own).